include/tune_quda.h

```diff
@@ -610,6 +611,21 @@ namespace quda {
    */
   void setPolicyTuning(bool);
 
+  /**
+   * @brief Query whether we are currently tuning a policy
```

Comment:
Just for cleanliness/consistency, is there a reason why we couldn't add this routine to the base `Tuneable` class instead and let it get inherited downstream?

Reply:
Not sure I follow. This is a global property here...

Reply:
Oh, yep, this is my bad, and what happens when I assume too much about what github's web interface squishes away.